backend/server.js

```diff
@@ -31,15 +31,16 @@ app.use(cors({
 
 app.use(session({
   // temp secret, will be changed and moved to env file
-  secret: 'foobar',
-  resave: false,
-  saveUninitialized: false,
+  secret: 'secretCode',
```

Comment:
Not 100% sure what the secret is used for or its importance, but maybe a long hash might be better for security purposes?